### ISSUE TITLE
A11y: Focus Indicator has insufficient contrast in variant product page elements

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5244]
The Focus indicator for: 

* Image Gallery pagination
* Variant property radio options

Have insufficient contrast against both the active element color and the background.

The focus indicator for:

* ‘Add to Shopping Cart’ button 

Has insufficient contrast against the background (it’s sufficient against the button color).

{panel:bgColor=#deebff}
This also affects the QuickView for CMS Extensions
{panel}

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39220**



[created from NEXT-39220, comment 506155]
